src/Tizen.NUI/src/public/BaseComponents/ViewAccessibilityEnum.cs

```diff
@@ -1456,4 +1456,23 @@ public enum AccessibilityTextBoundary
         [EditorBrowsable(EditorBrowsableState.Never)]
         Paragraph,
     }
+
+    /// <summary>
+    /// Accessibility coordinate type describing if coordinates are relative to screen or window
+    /// </summary>
+    /// <seealso cref="View.AccessibilityGetRangeExtents" />
+    [EditorBrowsable(EditorBrowsableState.Never)]
+    public enum AccessibilityCoordinateType
+    {
+        /// <summary>
+        /// Specifies xy coordinates relative to the screen.
+        /// </summary>
+        [EditorBrowsable(EditorBrowsableState.Never)]
+        SCREEN,
```

Comment:
Looking at other enums, it should probably be PascalCase i.e. `Screen` and `Window`.

Reply:
Oops. Please hide me.